apps/web-mzima-client/src/app/shared/components/language/language.component.html

```diff
@@ -7,7 +7,7 @@
     >
       <mat-select-trigger class="language__selected">
         {{ selectedLanguage }}
-        <mat-icon svgIcon="arrow-down" class="language__selected__arrow"></mat-icon>
+        <!-- <mat-icon svgIcon="arrow-down" class="language__selected__arrow"></mat-icon> -->
       </mat-select-trigger>
```

Comment:
Instead of commenting out the code, remove it entirely. If we want to go back and see a change, we can always look at the git-history :)